test/hack/cleanup/main.go

```diff
@@ -169,16 +176,29 @@ func (sg *securitygroup) Type() string {
 	return "SecurityGroup"
 }
 
-func (sg *securitygroup) Get(ctx context.Context, expirationTime time.Time) (ids []string, err error) {
+func (sg *securitygroup) Get(ctx context.Context, clusterName string, expirationTime time.Time) (ids []string, err error) {
 	var nextToken *string
+	var ec2Filter []ec2types.Filter
+
+	if clusterName == "" {
+		ec2Filter = []ec2types.Filter{
```

Comment:
We can't clean-up all the security groups by tag when we are cleaning up at the end of a test? That might be more reliable long-term than relying on name, which we are going to forget to update if we ever change that test

Reply:
I'm not getting what you mean here

Reply:
It looks like we are just looking for a single security group name here and I was wondering if we could make it so that the security groups are tagged a certain way during the test so that we can delete them off of their tag vs. off a specific name that might change over time

Reply:
The Idea was to delete only the SG from our drift test if that is being orphaned